content/en/blog/_posts/2021-11-12-are-you-ready-for-dockershim-removal/index.md

```diff
@@ -3,9 +3,11 @@ layout: blog
 title: "Dockershim removal is coming. Are you ready?"
 date: 2021-11-12
 slug: are-you-ready-for-dockershim-removal
+author: >
+   Sergey Kanzhelev (Google)
 ---
 
-**Authors:** Sergey Kanzhelev, Google. With reviews from Davanum Srinivas, Elana Hashman, Noah Kantrowitz, Rey Lejano.
+**Reviewers:** Davanum Srinivas, Elana Hashman, Noah Kantrowitz, Rey Lejano.
```

Comment:
Hmm. We could add a new front matter field for this editorial role. Maybe in a follow-up PR?